proto/rill/runtime/v1/queries.proto

```diff
@@ -1024,6 +1024,8 @@ message TimeRangeSummary {
   google.protobuf.Timestamp min = 1;
   // Not optional, not null
   google.protobuf.Timestamp max = 2;
+  // Not optional, not null
+  google.protobuf.Timestamp watermark = 4;
   message Interval {
```

Comment:
Do you use the `Interval` on the frontend? Or can we just remove it?

Reply:
No we do not. Safe to remove it.

Reply:
We are doing a `max-min` in UI. There was one place in queries where it was used. But it is better to move the math to that query.